pkg/combustion/templates/13b-add-users.sh.tpl

```diff
@@ -4,20 +4,27 @@ set -euo pipefail
 # Without this, the script will run successfully during combustion, but when /home
 # is mounted it will hide the /home used during these user creations.
 mount /home
-
+#---
 {{- range $user := . }}
-
 {{- /* Non-root users */}}
 {{- if (ne $user.Username "root") }}
-PRIMARY_GROUP=""
```

Comment:
This diff is ugly because I changed the approach to improve testability. Environment variables worked, but it was much more difficult to unit test the result since the variables themselves are reused and the `useradd` call was always the same (simply referencing the env variables). This approach produces a bespoke `useradd` command per user which is both safer (so we don't run the risk of not properly resetting the env variables in the loop) and significantly easier to test.